src/de/tuebingen/uni/ub/hc/Corpus/IxTheoCorpus.java

```diff
@@ -0,0 +1,264 @@
+package de.tuebingen.uni.ub.hc.Corpus;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.Vector;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Vector;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import de.tuebingen.uni.ub.hc.enums.IxTheo_Annotation;
+import edu.stanford.nlp.ling.CoreAnnotations.NamedEntityTagAnnotation;
+import edu.stanford.nlp.ling.CoreAnnotations.PartOfSpeechAnnotation;
+import edu.stanford.nlp.ling.CoreLabel;
+
+/**
+ * Corpus class holds all individual files (Titeldatensaetze) in a list
+ * 
+ * @author heike cardoso
+ *
+ */
+public class IxTheoCorpus implements Serializable {
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = -5152829236449977331L;
+	private Vector<IxTheoRecord> fileList;
+	private HashMap<IxTheo_Annotation, Integer> ixTheoAnnoCount;
+	private HashMap<String, Integer> wordCounts, lemmaCounts, neCounts;
+	private Vector<String> lemmaStringVector, neStringVector;
+
+	public IxTheoCorpus() {
+		fileList = new Vector<IxTheoRecord>();
+		setWordCounts(new HashMap<String, Integer>());
+		setLemmaCounts(new HashMap<String, Integer>());
+		setNeCounts(new HashMap<String, Integer>());
+		setIxTheoAnnoCount(new HashMap<IxTheo_Annotation, Integer>());
+		setLemmaStringVector(new Vector<String>());
+		setNeStringVector(new Vector<>());
+	}
+
+	public Vector<IxTheoRecord> getFileList() {
+		return fileList;
+	}
+
+	public void setFileList(Vector<IxTheoRecord> fileList) {
```

Comment:
You only use this in the constructor.  Please remove the function  and move the one-line function body into the constructor.